admittance_controller/src/admittance_controller.cpp

```diff
@@ -553,8 +559,8 @@ void AdmittanceController::write_state_to_hardware(
 {
   // if any interface has nan values, assume state_commanded is the last command state
   size_t pos_ind = 0;
-  size_t vel_ind = pos_ind + has_velocity_command_interface_;
-  size_t acc_ind = vel_ind + has_acceleration_state_interface_;
+  size_t vel_ind = (has_position_command_interface_) ? has_velocity_command_interface_ : pos_ind;
```

Comment:
Can you please explain to me the logic here? 
It might be missing `pos_ind +`?

Thank you!